adaptive/build.gradle.kts

```diff
@@ -0,0 +1,134 @@
+android {
+    namespace = "com.google.accompanist.adaptive"
+
+    compileSdk = 33
+
+    defaultConfig {
+        minSdk = 21
+        // targetSdkVersion has no effect for libraries. This is only used for the test APK
+        targetSdk = 33
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    buildFeatures {
+        buildConfig = false
+        compose = true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = libs.versions.composeCompiler.get()
+    }
+
+    lint {
+        textReport = true
+        textOutput = File("stdout")
+        // We run a full lint analysis as build part in CI, so skip vital checks for assemble tasks
+        checkReleaseBuilds = false
+    }
+
+    packagingOptions {
+        // Some of the META-INF files conflict with coroutines-test. Exclude them to enable
+        // our test APK to build (has no effect on our AARs)
+        resources {
+            excludes += listOf("/META-INF/AL2.0", "/META-INF/LGPL2.1")
+        }
+    }
+
+    testOptions {
+        unitTests {
+            isIncludeAndroidResources = true
+        }
+        animationsDisabled = true
+    }
+
+    sourceSets {
+        named("test") {
+            java.srcDirs("src/sharedTest/kotlin")
+            res.srcDirs("src/sharedTest/res")
+        }
+        named("androidTest") {
+            java.srcDirs("src/sharedTest/kotlin")
+            res.srcDirs("src/sharedTest/res")
+        }
+    }
+}
+
+metalava {
+    sourcePaths.setFrom("src/main")
+    filename.set("api/current.api")
+    reportLintsAsErrors.set(true)
+}
+
+dependencies {
+    api(libs.compose.foundation.foundation)
+    api(libs.compose.ui.ui)
+    api(libs.androidx.window)
+
+    implementation(libs.napier)
+    implementation(libs.kotlin.coroutines.android)
+
+    // ======================
+    // Test dependencies
+    // ======================
+
+    androidTestImplementation(project(":internal-testutils"))
+    testImplementation(project(":internal-testutils"))
+
+    androidTestImplementation(project(":testharness"))
+    testImplementation(project(":testharness"))
+
+    androidTestImplementation(libs.junit)
+    testImplementation(libs.junit)
+
+    androidTestImplementation(libs.truth)
+    testImplementation(libs.truth)
+
+    androidTestImplementation(libs.compose.ui.test.junit4)
+    testImplementation(libs.compose.ui.test.junit4)
+
+    androidTestImplementation(libs.compose.ui.test.manifest)
+    testImplementation(libs.compose.ui.test.manifest)
+
+    androidTestImplementation(libs.androidx.test.runner)
+    testImplementation(libs.androidx.test.runner)
+
+    androidTestImplementation(libs.androidx.window.testing)
+    testImplementation(libs.androidx.window.testing)
+
+    testImplementation(libs.robolectric)
+}
+
+apply(plugin = "com.vanniktech.maven.publish")
```

Comment:
Lets use the plugins block https://developer.android.com/build/migrate-to-kotlin-dsl#migrate-buildscript